media/css/base/page-not-found.scss

```diff
@@ -2,79 +2,68 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../pebbles/includes/lib';
-@import '../hubs/sections';
+@import '../../protocol/css/includes/lib';
 
-.mzp-c-navigation + #outer-wrapper,
-.moz-global-nav-drawer + #outer-wrapper {
-    padding-top: 40px;
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+.no-highlight:link, .no-highlight:visited, .no-highlight:hover, .no-highlight:active {
+    color: black;
 }
 
 main {
-    min-height: 450px;
-
-    header {
-        margin-bottom: 40px;
+    font-size: 18px;
+}
 
-        h1 {
-            @include font-size-huge;
-            color: #fff;
-            line-height: 1.3;
+h2 {
```

Comment:
This is a pretty broad selector and will apply to anything on the page. It may be ok as is, but should we add any more content in the future it could have unintended consequences. Suggest using something more specific to the page, like `.500-header`?